class-form.php

```diff
@@ -76,11 +134,71 @@ public function get_fields() {
 		 * @param array $content The content value.
 		 */
 		public function load_args_from_layout_content( $content ) {
-
 			$this->heading = $content['heading'];
-			$this->form = $content['form_value'];
+			$this->form    = $this->get_form_html( $content['form_value'] );
```

Comment:
`public $form;` sier ingenting om hva man kan forvente av innhold i variablen. Tidligere var dette form id, nå ser det ut til å være html output. Jeg foreslår at du innfører `$form_id` og `$form_plugin` (gf, cf, nf) og endrer `$form` til `$form_html`.

Reply:
Trenger vi dette på lagret på objektet eller kun i funksjonen `get_form_html()` for bedre lesbar kode?